packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1246,6 +1251,28 @@
                     "required": false
                 }
             ]
+        },
+        {
+            "name": "toolkit_trackScenario",
```

Comment:
this was upstreamed to https://github.com/aws/aws-toolkit-common/pull/856 , so should not be here. if there are any changes, then please upstream them.

Reply:
Somehow the rebased version replaced the change adding the previous trackScenario. I'll update again.